src/org/gk/elv/ElvReactionEditHelper.java

```diff
@@ -239,6 +239,7 @@ private Renderable getRenderableForPE(GKInstance pe,
             rPe = zoomableEditor.getFreeFormObject(pe);
         if (rPe == null)
             rPe = zoomableEditor.insertInstance(pe);
+        System.out.println("rPe: " + rPe);
```

Comment:
Please delete this line in a production setting.

Reply:
My mistake — will do!